CONTRIBUTION.md

```diff
@@ -0,0 +1,245 @@
+
+# How to contribute
+
+## Submitting bugs
+
+### Due diligence
+-------------
+
+Before submitting a bug, please do the following:
```

Comment:
Sure, this would be redundant anyway considering we have a bug template when creating issues. We should make sure that the template there is easily understood and easy to use